pages/showcase.tsx

```diff
@@ -0,0 +1,296 @@
+                            'repeat(1, 1fr)',
+                            'repeat(2, 1fr)',
+                            'repeat(12, 1fr)',
+                          ]}
+                          templateRows={[
+                            'repeat(1, 1fr)',
+                            'repeat(1, 1fr)',
+                            'repeat(2, 1fr)',
+                          ]}
+                          rowGap={12}
+                          columnGap={8}
+                        >
+                          {items.map(({ name, image, url, github }, i) => {
+                            const colSpan = [1, 2, 1, i % 3 === 0 ? 6 : 3]
+                            const rowSpan = [1, 2, 2, i % 3 === 0 ? 2 : 1]
+
+                            if (image == null)
+                              return (
+                                <ShowcaseContainerGridItem
+                                  key={name}
+                                  colSpan={colSpan}
+                                  rowSpan={rowSpan}
+                                >
+                                  <Mask
+                                    name={name}
+                                    github={github}
+                                    url={url}
+                                    showMask={true}
+                                  />
+                                  <ChakraNextImage
+                                    height={478}
+                                    width={850}
+                                    borderRadius='5px'
+                                    layout='responsive'
+                                    src={`/og-image.png`}
+                                  />
+                                </ShowcaseContainerGridItem>
+                              )
+                            return (
+                              <ShowcaseContainerGridItem
+                                key={name}
+                                colSpan={colSpan}
+                                rowSpan={rowSpan}
+                              >
+                                <Mask name={name} github={github} url={url} />
+                                <ChakraNextImage
+                                  height={478}
+                                  width={850}
+                                  borderRadius='5px'
+                                  layout='responsive'
+                                  src={`/${image}`}
+                                />
+                              </ShowcaseContainerGridItem>
+                            )
+                          })}
+                        </Grid>
+                      </TabPanel>
+                    )
+                  }),
+                [],
+              )}
+            </TabPanels>
+          </Tabs>
+        </Box>
+        <DiscordStrip />
+        <Footer />
+      </Box>
+    </>
+  )
+}
+
+const GithubIcon = (props: React.ComponentProps<'svg'>) => (
```

Comment:
We have react icons as dependency for icons we use in most places. Maybe it would make sence to use FaGithub from react-icons.

Reply:
Actually I just took the icon implemented in `header` to `showcase` page. 
I think maybe we should pull them out and create an individual component in next pr.

Reply:
Oh then i missed that one mb. I'd move it to its own component now otherwise we risk to forget about that :)

Reply:
Let's do that 😁